components/automate-ui/src/app/entities/notifications/notification.model.ts

```diff
@@ -1,9 +1,6 @@
 export enum Type {
   error = 'error',
   info = 'info',
-  /* TODO (tc) This is a stopgap because product decided it was
-     too risky to ship the full experience of a license banner.
-     This code will be removed first thing after ChefConf. */
```

Comment:
No longer relevant; license type is now being used by the license notification banner (license-notifications.component.html